### DISCUSSION TITLE
Adding roads to landscape maps

### DISCUSSION THREAD

**Discussion by JoshL22** [RangeShifter/RangeshiftR-tutorials#16] [Q&A] [answered]
I have an issue whereby when adding roads to my landscape rasters the roads become discontinous due to the pixel size. I have had to incrementally buffer my roads as shp files until a point when after conversion the roads are then continous. This leads to roads which are disproportionaltly wide. 

I know this isn't directly a rangeshifter program issue, nevertheless I thought it might still be relevant due to roads being notorious as barriers to dispersal. Is there any way to convert roads into a continous layer at a proposional size, in QGIS or R for example ? For reference my raster cells are 100m² in size, and I can't go smaller than that for my model. 

Thanks for the help again, 

Josh

**Comment by damariszurell** [comment, marked as answer]
Dear Josh,
As this is a GIS and not a RangeShiftR issue, we kindly recommend to consult relevant GIS literature.
Best,
Damaris

**Comment by JoshL22** [comment]
Dear Damaris, 

Ok, yes sorry for misplacing the question. Thanks

Josh
